assets/sass/components/global/_googlesitekit-entity-header.scss

```diff
@@ -0,0 +1,122 @@
+/**
+ * Entity Header Banner styles.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-entity-header {
+		background-color: map-get($wp-colors, "gray-2");
+		color: $c-cape-cod;
+		display: flex;
+		justify-content: space-between;
+		padding: 16px;
+		-webkit-position: sticky;
+		position: sticky;
+		top: 132px;
+		z-index: 9; // should be under _googlesitekit-dashboard-navigation
+
+		body.admin-bar & {
+
+			@media ( min-width: $width-tablet + 1 + px ) {
+				top: 194px;
+			}
+
+			@media ( min-width: $bp-wpAdminBarTablet ) {
+				top: 180px;
+			}
+
+			@media ( min-width: $bp-desktop ) {
+				top: 196px;
+			}
+		}
+
+		@media (min-width: $bp-tablet) {
+			padding: 16px 34px;
+		}
+	}
+
+	.googlesitekit-entity-header--scrolled {
+		box-shadow: 0 3px 3px rgb(0 0 0 / 10%);
+		transition: box-shadow 0.3s;
+	}
+
+	.googlesitekit-entity-header__back {
+		font-size: 12px;
+		font-weight: $fw-primary-normal;
+		height: auto;
+		letter-spacing: 0.3px;
+		line-height: 1;
+		margin: 4px 0 0;
+		min-width: auto;
+		text-transform: none;
+
+		@media (min-width: $bp-tablet) {
+			font-size: 16px;
+			margin: 4px 0 12px;
```

Comment:
I think the bottom margin should be equal `4px` too because currently the back button looks vertically aligned incorrectly.

![image](https://user-images.githubusercontent.com/873049/145573476-a0cc4011-8392-4bae-b22f-66b06c401110.png)